### ISSUE TITLE
Timeout waiting for SSH

### ISSUE THREAD

**Issue by scrapbird** [pixelcog/opsworks-vm#13]
When trying to run `rake build install`:

> ==> Rake: Building ubuntu1404 for virtualbox
> rm -f build/ubuntu1404-opsworks-virtualbox.box && packer build -only=virtualbox-iso template/ubuntu1404.json
> virtualbox-iso output will be in this color.
> 
> ==> virtualbox-iso: Downloading or copying Guest additions
>     virtualbox-iso: Downloading or copying: file:///Applications/VirtualBox.app/Contents/MacOS/VBoxGuestAdditions.iso
> ==> virtualbox-iso: Downloading or copying ISO
>     virtualbox-iso: Downloading or copying: http://releases.ubuntu.com/14.04/ubuntu-14.04.3-server-amd64.iso
>     virtualbox-iso: Download progress: 0%
>     virtualbox-iso: Download progress: 1%
>     virtualbox-iso: Download progress: 2%
>     virtualbox-iso: Download progress: 4%
>     virtualbox-iso: Download progress: 7%
>     virtualbox-iso: Download progress: 10%
>     virtualbox-iso: Download progress: 13%
>     virtualbox-iso: Download progress: 16%
>     virtualbox-iso: Download progress: 19%
>     virtualbox-iso: Download progress: 22%
>     virtualbox-iso: Download progress: 25%
>     virtualbox-iso: Download progress: 28%
>     virtualbox-iso: Download progress: 31%
>     virtualbox-iso: Download progress: 34%
>     virtualbox-iso: Download progress: 37%
>     virtualbox-iso: Download progress: 40%
>     virtualbox-iso: Download progress: 41%
>     virtualbox-iso: Download progress: 43%
>     virtualbox-iso: Download progress: 45%
>     virtualbox-iso: Download progress: 48%
>     virtualbox-iso: Download progress: 51%
>     virtualbox-iso: Download progress: 51%
>     virtualbox-iso: Download progress: 51%
>     virtualbox-iso: Download progress: 52%
>     virtualbox-iso: Download progress: 53%
>     virtualbox-iso: Download progress: 54%
>     virtualbox-iso: Download progress: 56%
>     virtualbox-iso: Download progress: 58%
>     virtualbox-iso: Download progress: 60%
>     virtualbox-iso: Download progress: 63%
>     virtualbox-iso: Download progress: 65%
>     virtualbox-iso: Download progress: 68%
>     virtualbox-iso: Download progress: 71%
>     virtualbox-iso: Download progress: 74%
>     virtualbox-iso: Download progress: 77%
>     virtualbox-iso: Download progress: 80%
>     virtualbox-iso: Download progress: 81%
>     virtualbox-iso: Download progress: 82%
>     virtualbox-iso: Download progress: 83%
>     virtualbox-iso: Download progress: 84%
>     virtualbox-iso: Download progress: 85%
>     virtualbox-iso: Download progress: 86%
>     virtualbox-iso: Download progress: 86%
>     virtualbox-iso: Download progress: 87%
>     virtualbox-iso: Download progress: 88%
>     virtualbox-iso: Download progress: 90%
>     virtualbox-iso: Download progress: 91%
>     virtualbox-iso: Download progress: 92%
>     virtualbox-iso: Download progress: 93%
>     virtualbox-iso: Download progress: 95%
>     virtualbox-iso: Download progress: 98%
> ==> virtualbox-iso: Starting HTTP server on port 8045
> ==> virtualbox-iso: Creating virtual machine...
> ==> virtualbox-iso: Creating hard drive...
> ==> virtualbox-iso: Creating forwarded port mapping for SSH (host port 4378)
> ==> virtualbox-iso: Starting the virtual machine...
>     virtualbox-iso: WARNING: The VM will be started in headless mode, as configured.
>     virtualbox-iso: In headless mode, errors during the boot sequence or OS setup
>     virtualbox-iso: won't be easily visible. Use at your own discretion.
> ==> virtualbox-iso: Waiting 5s for boot...
> ==> virtualbox-iso: Typing the boot command...
> ==> virtualbox-iso: Waiting for SSH to become available...
> ==> virtualbox-iso: Timeout waiting for SSH.
> ==> virtualbox-iso: Unregistering and deleting virtual machine...
> ==> virtualbox-iso: Deleting output directory...
> Build 'virtualbox-iso' errored: Timeout waiting for SSH.
> 
> ==> Some builds didn't complete successfully and had errors:
> --> virtualbox-iso: Timeout waiting for SSH.
> 
> ==> Builds finished but no artifacts were created.
> rake aborted!
> Command failed with status (1): [rm -f build/ubuntu1404-opsworks-virtualbox...]
> /Users/brentonmorris/Software/opsworks-vm/Rakefile:47:in `build_box'
> /Users/brentonmorris/Software/opsworks-vm/Rakefile:9:in`block (3 levels) in <top (required)>'
> Tasks: TOP => build => virtualbox:build
> (See full trace by running task with --trace)

Seems like the box is still installing the OS and packer times out waiting for SSH, then deletes the box and closes.

Edit: I should add that I am running OSX 10.11.2 and have the latest homebrew versions of vagrant, rake and packer.

**Comment by matejvelikonja** [comment]
I've encountered the same issue. I've fixed it with increasing SSH timeout in json file.

Add this line `"ssh_wait_timeout": "3600s",` to file https://github.com/pixelcog/opsworks-vm/blob/ecbf7523d0d3cc192fb1985c4203cdfce62787cb/template/ubuntu1404.json

**Comment by scrapbird** [comment]
Thanks for that @matejvelikonja, I ended up running it in debug mode so that it would pause after each step to give it a little more time. It would be nice if it tried to ssh in at intervals or something so it wouldn't matter how long the install takes.

**Comment by ghost** [comment]
I got same here 

```
==> virtualbox-iso: Timeout waiting for SSH.
==> virtualbox-iso: Unregistering and deleting virtual machine...
==> virtualbox-iso: Deleting output directory...
Build 'virtualbox-iso' errored: Timeout waiting for SSH.

==> Some builds didn't complete successfully and had errors:
--> virtualbox-iso: Timeout waiting for SSH.

==> Builds finished but no artifacts were created.
rake aborted!
```

**Comment by ghost** [comment]
Please also see [pull request 19](https://github.com/pixelcog/opsworks-vm/pull/19)

**Comment by scrapbird** [comment]
@Dual-EC-DRBG Looks good 👍

**Comment by mikegreiling** [comment]
Fixed via PR #19 

thanks!
